src/plugin/ipc/ipc.cpp

```diff
@@ -93,6 +93,10 @@ static DmtcpBarrier socketBarriers[] = {
   { DMTCP_PRIVATE_BARRIER_PRE_CKPT, SocketConnList::saveOptions, "PRE_CKPT" },
   { DMTCP_LOCAL_BARRIER_PRE_CKPT, SocketConnList::leaderElection,
     "LEADER_ELECTION" },
+  { DMTCP_GLOBAL_BARRIER_PRE_CKPT, SocketConnList::ckptRegisterNSData,
+    "CKPT_NS_REGISTER_DATA" },
```

Comment:
Can we rename this to `CKPT_REGISTER_PEER_INFO` or something like that so it's easy to decipher what it's doing?